CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Change Log
 This file documents all notable changes to juttle-client-library. The release numbering uses [semantic versioning](http://semver.org).
 
+## 0.4.0
+
+## Major Changes
+
+  Added an `edit` button to enable editing code in the browser [#29](https://github.com/juttle/juttle-viewer/issues/29)
```

Comment:
nit: seems better to say we added an `edit` mode, not just a button. ditto for debug.